feflow/tests/test_nonequilibrium_cycling.py

```diff
@@ -295,6 +296,7 @@ def test_create_execute_gather(
         # print(f"Free energy = {fe_estimate} +/- {fe_error}") # DEBUG
 
     @pytest.mark.gpu_ci
+    @pytest.mark.slow
```

Comment:
We were already running these tests in CI before, so they aren't THAT slow. We should not mark them, imo.
```suggestion
```